sci-biology/ants/files/99ants

```diff
@@ -0,0 +1 @@
+ANTSPATH=/usr/bin/
```

Comment:
yes, sadly it is. Some ANTs scripts check if the variable is set when they are called and fail otherwise.